desktop/core/src/desktop/js/apps/editor/components/resultGrid/ko.resultDownloadModal.js

```diff
@@ -72,7 +72,8 @@ class DownloadResultModal {
 
     params.executable.toContext(id).then(jsonContext => {
       this.cookieId = 'download-' + id;
-      $.cookie(this.cookieId, null, { expires: -1, path: '/' });
+      // eslint-disable-next-line no-undef
+      Cookies.set(this.cookieId, null, { expires: -1, path: '/' });
```

Comment:
Missing import statement (`import Cookies from 'js-cookie'`). Once the import is in place you can remove the eslint-disable lines